data/scenarios/tutorial.sav

```diff
@@ -347,10 +350,10 @@ function tutorial_tech_researched_callback(tech, player, reason)
   end
   if numtechs == 0 then
     notify.event(player, nil, E.SCRIPT,
-_("First tech researched: You have gained your first technology! Technological advances are an integral concept in Freeciv21. As you increase your technology level new units, buildings, governments, and various free bonuses will become available to you. For now, go over into the Research report and play around a little. Press F6 to open the Research report [or click on the Research tab]. Find the technology called Republic and click on it. Now Republic has been set as your technology goal; this means the next advance on the path toward this technology will automatically be chosen to research. In the top of the window it shows how much scientific research you are accomplishing each turn -- research (bulbs) comes from the taxable trade in your cities. In the bottom of the window is the tech tree; here you may left-click to set the research target or research goal. You can also hover your mouse to get help about a particular technology. When you are done playing around click on the View tab (F1) to return to the map view."))
+_("First tech researched: You have gained your first technology! Technological advances are an integral concept in Freeciv21. As you increase your technology level new units, buildings, governments, and various free bonuses will become available to you. For now, go over into the Research View and play around a little. Press F6 to open the Research View [or click on the Research button on the top bar]. Find the technology called Republic and click on it. Now Republic has been set as your technology goal; this means the next advance on the path toward this technology will automatically be chosen to research. In the top of the window it shows how much scientific research you are accomplishing each turn -- research (bulbs) comes from the taxable trade in your cities. In the bottom of the window is the tech tree; here you may left-click to set the research target or research goal. You can also hover your mouse to get help about a particular technology. When you are done playing around click on the Map View button (F1) to return to the map view."))
```

Comment:
Typo: missing comma after "level" in "As you increase your technology level new units, buildings, ..."

Consider breaking up "Now Republic has been set as your technology goal; this means the next advance..." into two sentences on the semicolon

Can we use an em dash instead of a double hyphen?

Consider breaking up "In the bottom of the window is the tech tree; here you may ..." into two sentences on the semicolon or joining them with a comma instead of a semicolon